client/src/views/About.vue

```diff
@@ -113,6 +132,28 @@ export default {
         }
       });
     },
+    help(){
+      window.open("http://localhost:3000/help#section-one", '_blank')
```

Comment:
this will probably only open on local systems. can you use a relative path? or use the baseService/unoService to declare this action?